views/administration/forms/form-classes-edit.hbs

```diff
@@ -5,7 +5,7 @@
 	<input type="hidden" name="referrer" value="{{../referrer}}" />
 
 	<a class="d-block mb-1" href="/administration/classes/{{../class/_id}}/manage"
-		title="Klasse '{{../class.name}}' verwalten.">
+		title="Klasse '{{../class.name}}' verwalten." data-testid="go-to-manage-class-btn">
```

Comment:
"go-to" does not tell much, if you really wana have something more specific I would go with `manage-class-btn`

Reply:
Done!